packages/verified-fetch/src/verified-fetch.ts

```diff
@@ -388,10 +464,35 @@ export class VerifiedFetch {
         this.log.error('Error parsing content type', err)
       }
     }
-
+    this.log.trace('setting content type to "%s"', contentType)
     response.headers.set('content-type', contentType)
   }
 
+  private checkForInvalidRangeRequest ({ offset, length, resource, stat }: { offset?: number, length?: number, resource: string, stat: UnixFSStats }): Response | undefined {
```

Comment:
I was able to move out some of the code from the handleDagPb, but could probably move this function outside of `verified-fetch` altogether.

Reply:
It could be made a function in the util folder?  No strong opinion.

Reply:
this function goes away with the addition of the byte-range-context class